src/components/AddList.js

```diff
@@ -0,0 +1,31 @@
+import './AddList.css';
+import {useState} from 'react';
+
+export default function AddList(props) {
+    const [text, setText] = useState('')
+
+    function onSubmit (e) {
+        e.preventDefault()
+        if(!text) {
+            alert('Please add a list name.')
+            return
+        }
+        props.addList(text)
+        setText('')
+    }
+
+    return (
+        <form className='list-form' onSubmit={onSubmit}>
```

Comment:
No need to have this in a form.  Can just have your Add button be a < button> with an onClick handler.